rs/state_manager/src/lib.rs

```diff
@@ -3351,6 +3351,37 @@ impl StateManager for StateManagerImpl {
             .min(requested_height)
             .max(Height::new(1));
 
+        // Log how Consensus calls this API when it has some extra states to keep.
```

Comment:
Normally, these logs will only show up in a few rounds before and after the checkpoint height and `extra_heights_to_keep` just contains that height.  Thus there will be no log spam.

Reply:
https://github.com/dfinity/ic/pull/2223 has just been merged and will provide something to log.